woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1733,9 +1701,11 @@ def test_select_return_schema(sample_df):
      (["Datetime"], "datetime"),
      (["Unknown", "EmailAddress"], "string"),
      (["Categorical"], "category"),
-     (["BooleanNullable"], "boolean")]
+     (["Boolean", "BooleanNullable"], "boolean")]
 )
 def test_select_retains_column_order(ww_type, pandas_type, sample_df):
+    if ks and isinstance(sample_df, ks.DataFrame) and (pandas_type == "category" or pandas_type == "string"):
```

Comment:
We can clean up this logic a bit:
`if _is_koalas_dataframe(sample_df) and pandas_type in ["category", "string"]:`